src/parser/name_resolution.c

```diff
@@ -941,9 +941,6 @@ pt_bind_name_or_path_in_scope (PARSER_CONTEXT * parser, PT_BIND_NAMES_ARG * bind
       /* If pt_name in group by/ having, maybe it's alias. We will try to resolve it later. */
       if (!is_pt_name_in_group_having (in_node))
 	{
-	  /* it may be a naked parameter or a path expression anchored by a naked parameter. Try and resolve it as
-	   * such. */
-	  node = pt_bind_parameter_path (parser, in_node);
 
 	  if (node == NULL && parser->flag.is_parsing_static_sql == 1)
```

Comment:
If the code at line 946 is removed, the `node == NULL` condition at line 948 will always be true.
This is because the condition for entering this block is when node is NULL.

Reply:
I see. The condition seems redundant. I'm going to change as follow. Thank you.
```suggestion
	  if (parser->flag.is_parsing_static_sql == 1)
```